### DISCUSSION TITLE
Resize filter with responsive option

### DISCUSSION THREAD

**Discussion by matthieu2607** [Cecilapp/Cecil#1469] [Q&A] [answered]
Hi @ArnaudLigny,

It would be nice if we could with the "resize" filter generate a responsive image (with different sizes) as we have at the markdown level.

Good day,

Matthieu

**Comment by ArnaudLigny** [comment, marked as answer]
Hello @matthieu2607, did you try to pipe the `resize` and the [`html`](https://cecil.app/documentation/templates/#html) filter?

For example:

```twig
{{ asset('image.jpg')|resize(300)|html({alt: 'Image description'}, {responsive: true}) }}
```

Note: `{responsive: true}` is not required if the option is enabled globally through [assets configuration](https://cecil.app/documentation/configuration/#assets).

**Comment by matthieu2607** [comment]
Oh nice, it's great, it's run ;)

Thank you @ArnaudLigny
